### PR TITLE
Init Payment Delegation DB Relay Example Code

### PR DESCRIPTION
**Pull request by spacesailor24**
Corresponding docs guide for this code is [here](https://github.com/LIT-Protocol/docs/pull/300)

Instructions on running the example are in the [README](https://github.com/LIT-Protocol/developer-guides-code/blob/wyatt/payment-delegation-relayer/payment-delegation-db-relayer/nodejs/README.md)

## Pre-Merge Standard Guide Checklist

- [ ]  User story/motivation for the guide has been identified
- [ ]  User story/motivation is addressed by the guide
- [x] Prerequisite knowledge is noted at the beginning of the guide, with links to where to learn more if the user does not possess this knowledge
- [x]  Example code has contains all code and components required for a basic user to run it without implicit knowledge of other code/tools that need to be incorporated
- [x]  Example code runs without bugs
- [x]  No pseudocode
- [ ]  Full written explanations of all steps required to use a given feature (alongside code)
- [x]  Warnings for when code is only relevant to a given testnet/function/version/etc.
- [ ]  Documentation titles and categorization are clear for new users
- [x]  Guide and example code has been reviewed, followed, and successfully run by a peer who has not already worked on the guide/code and is taking on the persona of a target user for the user story/motivation
- [ ] Guide has been reviewed for correct spelling and grammar
- [ ]  Project Lead (or another knowledgeable party) for the feature has reviewed the guide/code and checked for accuracy/completeness
- [ ]  Final review and sign-off from Product Manager

cc @alexvotofuture

### CODE REVIEW

File: payment-delegation-db-relayer/nodejs/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Payment Delegation Database
+
+<!-- TODO Add link to docs once it's published -->
+
+This directory contains an example usage of the Payment Delegation Database to register a new payer wallet, and add users as payees. There is a corresponding Lit docs page that covers the code in this example which is available here.
```
Comment (alexvotofuture) [inline payment-delegation-db-relayer/nodejs/README.md:5]: Hyperlink?

File: payment-delegation-db-relayer/nodejs/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+3. Initialize the `.env` file by copy `.env.example`:
+   ```
+   cp .env.example .env
+   ```
+4. Fill out the `.env`
+   - For the `LIT_NETWORK` variable, the following options are available:
+     - `habanero`
+     - `manzano`
+     <!-- Add link to form -->
+   - For the `LIT_RELAYER_API_KEY` variable, if you don't already have a Relayer API key, you can fill out this form to request one.
```
Comment (alexvotofuture) [inline payment-delegation-db-relayer/nodejs/README.md:23]: Hyperlink to form?

**Comment by alexvotofuture** [comment]
Please confirm when the guide has been followed by another person (other than Joseph) and the code has been run without bugs on their end. After that I think we're good!

File: payment-delegation-db-relayer/nodejs/README.md (hunk 1 above)
Comment (spacesailor24) [reply payment-delegation-db-relayer/nodejs/README.md]: Added, link will just 404 until the docs PR is merged and published

**Review by a1ttech — APPROVED** [review]

**Review by a1ttech — APPROVED** [review]

**Pull request merged**